cli/pom.xml

```diff
@@ -101,6 +101,7 @@
                     <excludes combine.children="append">
                         <exclude>vendor/**</exclude>
                         <exclude>glide.*</exclude>
```

Comment:
Seems it's version dependent. I started getting the `.glide` folder, while ppl on older go releases (same glide version) are on `glide.*`.